protoc-gen-openapiv2/internal/genopenapi/template.go

```diff
@@ -2228,6 +2228,14 @@ func protoComments(reg *descriptor.Registry, file *descriptor.File, outers []str
 			// - join by \n
 			comments = strings.Replace(comments, "\n ", "\n", -1)
 		}
+		if loc.TrailingComments != nil {
+			trailing := strings.TrimSpace(*loc.TrailingComments)
+			if comments == "" {
+				comments = trailing
+			} else {
+				comments += "\n\n" + trailing
```

Comment:
Is there a good reason to include double newlines between these? I kinda feel like a single newline might be enough?

Reply:
There are two points:
1. To distinguish the title and the description, when there is a leading comment without dot and a trailing comment
2. To put the trailing comment in separate paragraph. This is work for [Redoc](https://github.com/Redocly/redoc) (I think it use markdown style rendering), but not [Swagger UI](https://swagger.io/tools/swagger-ui/) (it just outputs string in spec yaml)